File: downloadablebookandexercisecode/1256/code_files/Chapter_01_USACO_Bronze/USACO_2016_jan_b1_Promote.py
# http://usaco.org/index.php?page=viewproblem2&cpid=591

f_in = open('promote.in', 'r')

b0, b1 = map(int, f_in.readline().split())
s0, s1 = map(int, f_in.readline().split())
g0, g1 = map(int, f_in.readline().split())
p0, p1 = map(int, f_in.readline().split())

# promoted

pm = p1 - p0
gm = g1 - g0 + pm
sm = s1 - s0 + gm

f_out = open('promote.out', 'w')
str1 = str(sm) + '\n' + str(gm) + '\n' + str(pm) + '\n'
f_out.write(str1)

File: downloadablebookandexercisecode/1256/code_files/Chapter_01_USACO_Bronze/USACO_2016_jan_b1_promote.cpp
// http://usaco.org/index.php?page=viewproblem2&cpid=591

#include <iostream>

using namespace std;

int main(){

    freopen("promote.in","r",stdin);
    freopen("promote.out","w",stdout);


    int b0, b1, s0, s1, g0, g1, p0, p1;
    cin >> b0 >> b1 >> s0 >> s1 >> g0 >> g1 >> p0 >> p1;

    int pm, gm, sm;
    pm = p1 - p0;
    gm = g1 - g0 + pm;
    sm = s1 - s0 + gm;

    cout << sm << endl;
    cout << gm << endl;
    cout << pm << endl;

}
